unit/suites/test_reconnect.py

```diff
@@ -66,6 +68,112 @@ def test_02_wrong_auth(self):
         con.close()
         self.srv.stop()
 
+    def test_03_mesh(self):
+        # Multiple servers are not supported on Windows
+        if os.name == 'nt':
+            return
+
+        # Start two servers
```

Comment:
Much of setup / teardown work in those tests are the same. I propose to move them into its own test suite and move common code into setUp() and tearDown() functions.